frontend/src/api/open-hands-axios.ts

```diff
@@ -1,7 +1,8 @@
 import axios from "axios";
 
-export const openHands = axios.create();
-
+export const openHands = axios.create({
+  baseURL: `http://${import.meta.env.VITE_BACKEND_BASE_URL || window?.location.host}`,
```

Comment:
Do we need the protocol? I don't think `window.location.host` has it. Maybe we need `origin`?
<img width="387" alt="Screenshot 2025-03-18 at 12 43 56 PM" src="https://github.com/user-attachments/assets/c963eb2e-3cc5-4696-8d7a-44ce8b683fa9" />

Reply:
If you add the protocol, it will be necessary to add a configuration for it, such as BACKEND_PROTOCOL, or modify VITE_BACKEND_BASE_URL, which may cause other impacts on the WebSocket.

`\openhands-db\frontend\src\context\ws-client-provider.tsx`
Line: 173
```js
    const baseUrl =
      import.meta.env.VITE_BACKEND_BASE_URL || window?.location.host;
```

What change should I make?

Reply:
It looks like axios expects the protocol: https://axios-http.com/docs/config_defaults

So you need this to be
```
baseURL: `${window.location.protocol}//${import.meta.env.VITE_BACKEND_BASE_URL || window?.location.host}`,
```